examples/model_evaluation/plot_train_test_split.py

```diff
@@ -142,6 +142,19 @@
 print("When expliciting, with the small typo, are the `X_train`'s still the same?")
 print(np.allclose(X_train_explicit, X_train_explicit_inverted))
 
+# %%
+# Returning a dictionnray instead of positional arguments
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Can you add a sentence to explain the purpose of this paragraph please? It can be quite short, for instance "Sending the 4 arrays to skore's EstimatorReport can be done easily this an additional parameter: "